tests/test_plugin_karma.py

```diff
@@ -83,6 +83,8 @@ def test_chained_brackets(self):
         self.bot.dispatch(':root@localhost PRIVMSG #chan :(foo--)++')
         self.assertEquals(self.bot.db[_format_key('foo--')]['up'], 1)
         with self.assertRaises(KeyError):
+            self.bot.db[_format_key('(foo')]
+        with self.assertRaises(KeyError):
```

Comment:
.. and then consistency would dictate similar behaviour for `test_chained` (i.e. `foo--++`). This is a deep rabbithole.